src/util/SHA1Hash.ts

```diff
@@ -0,0 +1,19 @@
+import * as sha1 from 'js-sha1'
+
+export default function calculateHash(data): string {
```

Comment:
Should be renamed to something like `calculateHashFromBase64`

Reply:
renamed